reports/models.py

```diff
@@ -238,10 +239,20 @@ class Test(models.Model):
     )
     name = models.TextField()
     testsuite = models.TextField()
-    env = models.TextField()
+    # this is a hash of the flags associated with this test
+    # users will use flags to distinguish the same test being run
+    # in a different environment
+    # for example: the same test being run on windows vs. mac
+    flags_hash = models.TextField()
```

Comment:
Oh interesting, is this the same flag as a codecov flag?

Reply:
This is a hash of the flags associated with the uploads that map to this test.

The idea is that users will use flags with their uploads to differentiate between the same test being run in a different environment (this used to be the env field but now it's the flags_hash field).

Reply:
Sounds good. I'd expect this field to be nullable too right?